config/locales/doorkeeper.en.yml

```diff
@@ -0,0 +1,152 @@
+en:
+  activerecord:
+    attributes:
+      doorkeeper/application:
+        name: 'Name'
+        redirect_uri: 'Redirect URI'
+    errors:
```

Comment:
Any specific reason why the `doorkeeper-i18n` gem is not used?

Reply:
I didn't come across it previously. I just followed Doorkeeper's getting started guide and they didn't mention anything about doorkeeper-i18n. Then again, do we need to do this? We'll remove these pages anyway, and it's only us admins who will use these pages. I don't think we need to put in too much effort for these temporary pages.